pkg/machine/qemu/virtiofsd.go

```diff
@@ -0,0 +1,107 @@
+//go:build linux || freebsd
+
+package qemu
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"time"
+
+	"github.com/containers/podman/v5/pkg/machine/define"
+	"github.com/containers/podman/v5/pkg/machine/vmconfigs"
+	"github.com/containers/storage/pkg/fileutils"
+	"github.com/sirupsen/logrus"
+)
+
+// We should probably make these bulid time configurable too
+const virtiofsdPathFedora = "/usr/libexec/virtiofsd"
+const virtiofsdPathDebian = "/usr/lib/qemu/virtiofsd"
```

Comment:
Ah man, and now that I look Arch Linux puts it in [`/usr/lib/virtiofsd`](https://archlinux.org/packages/extra/x86_64/virtiofsd).  Fun.  I guess we probably should make this build-time configuration.

Reply:
Maybe we should use config.FindHelperBinary(), that looks in /usr/libexec/podman and some other places.
I think we could then just do the podman packages do a symlink to /usr/libexec/virtiofsd on fedora and the other distros could create the correct symlink for their packages.

Reply:
Good idea!  That makes sense, it should work like `catatonit` for example.  Done.